### DISCUSSION TITLE
[Unintended Feature] Issue in color scheme

### DISCUSSION THREAD

**Discussion by SPyofgame** [squidfunk/mkdocs-material#5245] [Q&A] [answered]
The default and slate colors got affected by the primary.
Switching between default and slate, the transition makes it a bit flashy.
I didn't see that happened before (eg: #5210) so I think this is also unintended

![image](https://user-images.githubusercontent.com/62207095/227004381-d5504275-ae78-43d8-95d2-43ea81f96d29.png)
![image](https://user-images.githubusercontent.com/62207095/227004296-ecd4db80-62ab-4c10-86bc-616b8dde3f93.png)

**Comment by squidfunk** [comment, marked as answer]
I'm sorry, but I don't understand what you mean. The colors of the scheme buttons? They take the primary color, yes.

**Reply by SPyofgame** [reply]
Oh, I thought their colors depended on the scheme before.

And I also mean that it is a bit of flashing black and white if I switch the palette, is it normal ?

**Reply by squidfunk** [reply]
This only happens on our own documentation and only when using the color demo switcher, not the actual palette switch. However, I agree that it is annoying, so I went ahead and fixed it in ab23604fb.

**Reply by SPyofgame** [reply]
Oh, thank you
